content/node/leanpocket.md

```diff
@@ -0,0 +1,94 @@
+
+LeanPocket is an optimization to the core client to allow nodes to share resources, significantly reducing resource requirements. This is a non-consensus-breaking change to the client.
+
+<!-- Add charts here -->
+
+{{% notice style="info" %}}
+LeanPocket is specifically geared toward multiple nodes running on the same server. If you are running a single node, or have each of your nodes running on different server, LeanPocket will not have any benefit or effect.
+{{% /notice %}}
+
+## Setup
+
+To enable LeanPocket:
+
+1. Create a new `lean_nodes_keys.json` file inside your Pocket configuration directory (typically `.pocket`).
+
+2. Format the file with a JSON array of your node's private keys:
+
+   ```
+   [
+     { "priv_key": "<PRIVATE_KEY_1>" },
+     { "priv_key": "<PRIVATE_KEY_2>" }
+   ]
+   ```
+
+3. Set/update your validators:
+
+   ```
+   pocket accounts set-validators <path/to/lean_nodes_keys.json>
+   ```
+
+   {{% notice style="info" %}}
+   Note the plural form in the command. The option `set-validators` is distinct from `set-validator` and is only relevant for LeanPocket.
+   {{% /notice %}}
+
+4. Edit your node's `config.json` file to add `lean_pocket: "true";` in the `pocket_config` block.
+
+5. Restart your Pocket service.
+
+## Tips
+
+* Using `--forceSetValidators` as an argument when starting Pocket will ensure that your validators are updated every time the process starts. If using this argument, you won't have to use the `set-validators` command when you update your `lean_node_keys.json` file.
+* Do not use `--useCache` as an argument when starting Pocket, as it can delay restarting your node.
+* You can get a list of node with the `get-validators` command:
+
+  ```
+  pocket accounts get-validators
+  ```
+* You can add metadata to your `lean_node_keys.json` file for organizational purposes:
+
+  ```
+  [
+    {
+        "priv_key": "<PRIVATE_KEY_1>",
+        "node_name": "first example"
+    }
+  ]
+  ```
+
+## Testing
+
+<!-- What's the simplest way to test that LeanPocket is working? -->
+
+## Considerations and Risks
+
+* Because your nodes are commingling resources with LeanPocket, there is a risk of downtime on all of your nodes simultaneously. Depending on your risk for downtime, we recommend that you institute some form of failover system for your nodes, or perhaps divide your nodes into multiple LeanPocket-managed groups.
+* Since all LeanPocket nodes use the same configuration, it is not possible to serve different groups of chains on each individual node. Each node uses the same `chains.json` file.
+* If you are converting your node fleet into using fewer servers (or just one), you will want to ensure that your server open file limit (`ulimit` and `fd` on Linux-based systems) is sufficient, as these values still scale linearly with each additional node.
+* We don't recommend running Validator nodes via LeanPocket at this time, because of the increased importance of the consensus state on the network.
```

Comment:
💯

Reply:
Can we use stronger wording here? Have intel of potential "gamifying" using lean pocket to reduce their validator node mgmt overhead/costs but control 10-20% of consensus. 

> * We don't recommend running Validator nodes via LeanPocket at this time, because of the increased importance of the consensus state on the network.

Reply:
Jessica, can you elaborate on this potential intel a little bit more? The only reason why I suggest validators not to use LeanPOKT at this time is that there is a lack of tooling in the entire ecosystem to support the level of scale that LeanPOKT opens the doors for. Historically, running 100+ validators on one server was unimaginable to think about. The network will need time to adjust for better architectures and LeanPOKT provides those opportunities. **That is the only reason why.**

What do you mean by gamifying? Pocket is a proof of stake network and following that ideology, the cost is suppose to be reflected in your stake. We should not be suggesting wording around arbitrary "proof of work through man-hours and infrastructure costs". Validation is supposed to be cheap to run, it always should have been.

Reply:
Hey! Using "gamify" here in the sense not to a fault of lean pocket but describing the strategy of a large group of validators (250+) to be able to consolidate and, to your point, we don't have the tooling and would prefer to be more conservative about our recommendation here until that supportive tooling exists. DM me if you need more info too!

Reply:
@jessicadaugherty What do you think would constitute stronger wording? We can't prevent people from stacking validators, so we can only really say "please don't".

I'm happy to put anything down here, but I just don't have a good gut feeling on this at the moment.